README.md

```diff
@@ -9,16 +9,54 @@ https://zookeeper.ibinx.com/master/
 
 ### Requirements
 
-* PHP 5.6 or later with MySQL PDO driver
```

Comment:
Why are we no longer supporting PHP 5.6 or Apache 2.2?

Reply:
this was per your directions to mark re. setting up the new ZK box. also it is a good idea to keep up with the times.